pkg/lang/ir/editor.go

```diff
@@ -79,7 +79,7 @@ func (g Graph) generateJupyterCommand(workingDir string) []string {
 
 	if g.JupyterConfig.Password != "" {
 		cmd = append(cmd, "--NotebookApp.password", g.JupyterConfig.Password,
```

Comment:
Should we set the password here?

Reply:
Oh i want to rebase the 2 commits to 1 commits.